src/config/list-pages.json

```diff
@@ -0,0 +1,47 @@
+{
+    "pages": [
+        {
+            "path": "/blocks",
+            "name": "blocks",
+            "props": {
+              "namespace": "block",
```

Comment:
Just my opinion, show change the name "namespace" to "storeNamespace".
incase miss understand, `mosaic namespace`

Reply:
Good suggestion, agree